AGENTS.md

```
@@ -0,0 +1,163 @@
+it('應該建立 LogLayer 實例', () => {})
+it('當 NODE_ENV 為 production 時應使用生產環境傳輸', () => {})
+```
+
+### 環境變數隔離
+每個測試前後必須儲存/還原 `process.env`
+</testing>
+
+<architecture>
+## 專案架構
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)